File: app/src/main/java/com/yvrun/officeprocess/dagger/module/HomeBindModule.java
package com.yvrun.officeprocess.dagger.module;

import com.yvrun.officeprocess.core.primary.home.HomeFragment;
import com.yvrun.officeprocess.mvp.contract.HomeContract;
import com.yvrun.officeprocess.mvp.contract.TestContract;

import dagger.Binds;
import dagger.BindsInstance;
import dagger.Module;
import dagger.Provides;

@Module
public abstract class HomeModule {
    @Binds
    abstract HomeContract.IHomeView bindView(HomeFragment fragment);
}

File: kotlinappother/src/main/java/com/chengzhen/kotlinappother/mvp/dagger/module/HomeBindModule.kt
package com.chengzhen.kotlinappother.mvp.dagger.module

import com.chengzhen.kotlinappother.fragment.HomeFragment
import com.chengzhen.kotlinappother.mvp.contract.HomeContract

import dagger.Binds
import dagger.Module

@Module
abstract class HomeBindModule {
    @Binds
    internal abstract fun bindView(fragment: HomeFragment): HomeContract.IHomeView
}
